gef-ui/server/controllers/activities-controller/index.js

```diff
@@ -23,39 +24,50 @@ const mapPortalActivities = (portalActivities) =>
       facilityId,
       maker,
       checker,
+      scheduledCancellation,
     };
 
     return mappedActivity;
   });
 
+/**
+ * Retrieves portal activities for a specific deal and renders the application activity page.
+ *
+ * @param {Object} req - The request object.
+ * @param {Object} req.params - The request parameters.
+ * @param {string} req.params.dealId - The ID of the deal.
+ * @param {Object} req.session - The session object.
+ * @param {string} req.session.userToken - The user token.
+ * @param {Object} res - The response object.
+ * @returns {Promise<void>} - Renders the application activity page with the relevant data.
+ */
 const getPortalActivities = async (req, res) => {
   const { params, session } = req;
   const { dealId } = params;
   const { userToken } = session;
   const deal = await getApplication({ dealId, userToken });
```

Comment:
not a blocker, might be worth adding a new line here so we don't have a long list